src/docmanager/context.ts

```diff
@@ -108,8 +118,8 @@ class Context implements IDocumentContext {
    * #### Notes
    * This is a read-only property.
    */
-  get kernelSpecs(): IKernelSpecIds {
-    return this._manager.getKernelSpecs();
+  get kernelspecs(): IKernelSpecIds {
```

Comment:
That is the place that needs updating.  The command is called `jupyter kernelspec list`.